plottr/plot/mpl.py

```diff
@@ -292,6 +304,40 @@ def ppcolormesh_from_meshgrid(ax: Axes, x: np.ndarray, y: np.ndarray,
     ax.set_ylim(y.min(), y.max())
     return im
 
+def plot_as_linecuts(ax: Axes, x: np.ndarray, y: np.ndarray,
+                     z: np.ndarray, cmap, **kw):
+    
+    """Plot a 2D array as the individual linecuts
+    Input are the corresponding arrays from a 2D ``MeshgridDataDict``.
+
+    Will attempt to fix missing points in the coordinates.
+
+    :param ax: axes to plot the colormesh into.
+    :param x: x component of the meshgrid coordinates
+    :param y: y component of the meshgrid coordinates
+    :param z: data values
+    :returns: the image returned by `pcolormesh`.
```

Comment:
is this true? the function returns the line and also the ScalarMappable. also the return type of this function is missing :)

Reply:
I will check the added comments, I added those quickly without properly looking at them